pkgs/development/libraries/libtiff/default.nix

```diff
@@ -1,21 +1,12 @@
-{ lib, stdenv
-, fetchurl
-
-, pkg-config
-, cmake
-
-, zlib
-, libjpeg
-, xz
-}:
+{ lib, stdenv, fetchurl, cmake, pkg-config, libdeflate, libjpeg, xz, zlib }:
```

Comment:
I undid this change because it reduces the diff in this PR, but having args on one line is not better for review since GitHub displays line edits well (highlighting the changed part), and has no meaningful impact on merges since conflicts are rare and are not likely caused just by the changed args. The formatting of the args affects readability, and for me (and I think in general) one arg per line makes it worse by drawing attention from the important parts of the definition to the trivial (and duplicated in its body).

Reply:
> and has no meaningful impact on merges since conflicts are rare and are not likely caused just by the changed args.

Often merging the master and staging branches I have a different experience here. Its very annoying having to check the lists of arguments when there are many arguments. Here there aren't many, but solving merge conflicts when you have a whole block of arguments is not fun.